shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/segment/ddl/column/position/ColumnAfterPositionSegment.java

```diff
@@ -27,7 +28,7 @@
     
     private final String afterColumnName;
```

Comment:
Why do not you change it as `columnSegment`?

Reply:
the same as above